scripts/go-lint-all.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -euo 
```

Comment:
```suggestion
set -euo pipefile
```

I know we don't use a pipe in this script, but it is still good to set this to be sure we exit with a non zero exit code if something fails.
Did you remove it for a reason?

Reply:
I understand the reason now 😬.

Reply:
I understand the reason now. Weird.